arbnode/sequencer.go

```diff
@@ -72,6 +73,7 @@ var DefaultSequencerConfig = SequencerConfig{
 	MaxAcceptableTimestampDelta: time.Hour,
 	Forwarder:                   DefaultSequencerForwarderConfig,
 	QueueSize:                   1024,
+	QueueTimeout:                time.Second * 25,
```

Comment:
Doesn't this need to be lower than 15 seconds?

Reply:
:+1: